### DISCUSSION TITLE
Editing data from a source sheet from a transposed view

### DISCUSSION THREAD

**Discussion by pacien** [saulpw/visidata#2548] [Q&A]
Hello!

Starting with a "source" sheet, I open its transposed version
"source_T" using `Shift-T`.

How can I modify the data (editing cells, adding new rows) in the
"source" sheet from the "source_T" sheet, essentially treating the
latter as a view of the same table?

Editing cells with `e` on "source_T" seems to have no effect.
Same with adding new columns on "source_T", which I'd like to be
reflected as new rows in "source.

> note!
> Modifications made to rows on derived sheets will be reflected on the source sheets.
>
> - https://www.visidata.org/docs/edit/

**Comment by saulpw** [comment]
You are correct, @pacien; it's an oversight.   Should be fixed now on develop.

**Reply by pacien** [reply]
Thanks, I confirm that patch https://github.com/saulpw/visidata/commit/0d3d529e2e94c87406d5395c16096d630bb00db6 makes it possible to edit cells in the source sheet from the transposed sheet.

However, adding columns in the transposed sheet still doesn't add corresponding rows in the source sheet. Is that expected?
